assets/images/distinguishingMarks/face-overlay.svg

```diff
@@ -1,3 +1,10 @@
5NTEyQzIuMDUwMzMgMS42MTk3NSAxLjQ1Nzc1IDEuNzQ0MzMgMS4wMjExMyAxLjgzNzg5QzAuODAzMTA2IDEuODg0NjEgMC42MjMyODYgMS45MjM5MSAwLjQ5ODY2NyAxLjk1MTE3QzAuNDM2NDkyIDEuOTY0NzcgMC4zODcxNDEgMS45NzUxNCAwLjM1NDEzNiAxLjk4MjQyQzAuMzM4MDQzIDEuOTg1OTggMC4zMjU0MzQgMS45ODkzNSAwLjMxNzAyNyAxLjk5MTIxQzAuMzEzMDk1IDEuOTkyMDcgMC4zMDkzMyAxLjk5MjcxIDAuMzA3MjYxIDEuOTkzMTZMMC4zMDQzMzEgMS45OTQxNEMwLjE3NjM2OSAyLjAyMjM4IDAuMDQ3ODA4NSAxLjk0Njg5IDAuMDEwMzg2MSAxLjgyMTI5Qy0wLjAyNjg5MTQgMS42OTU1OSAwLjAzOTA4MTMgMS41NjE4MyAwLjE2MTc1MyAxLjUxNTYyTDAuMTY0NjgzIDEuNTE0NjVDMC4xNjY1NTYgMS41MTM5NSAwLjE2OTkzNSAxLjUxMzAyIDAuMTczNDcyIDEuNTExNzJDMC4xODExIDEuNTA4ODggMC4xOTMwNTQgMS41MDQ0MyAwLjIwNzY1MiAxLjQ5OTAyQzAuMjM3NTM2IDEuNDg3OTYgMC4yODIyODggMS40NzE3MSAwLjMzODUxMSAxLjQ1MTE3QzAuNDUxNDk0IDEuNDA5OTEgMC42MTUwNTMgMS4zNTA4NyAwLjgxNTA3NCAxLjI4MDI3QzEuMjE1MiAxLjEzOTA1IDEuNzY0OTYgMC45NTAyNDggMi4zNjE5NSAwLjc2MTcxOUMyLjk1ODQxIDAuNTczMzY0IDMuNjA1MjEgMC4zODM3MTEgNC4xOTg4NiAwLjI0MTIxMUM0Ljc4NzE1IDAuMTAwMDIyIDUuMzQzMjIgNi4yODI4MmUtMDUgNS43NDk2NCAwWiIgZmlsbD0iIzE5MjkzNiIvPgo8cGF0aCBkPSJNMzAuMTM3NCAwQzI5LjM2MDYgMCAyNy41ODg5IDAuMjU1MjI1IDI1Ljk2NTUgMC41NjczODNDMjUuMTQ5OSAwLjcyNDIyNyAyNC4zNjEyIDAuODk3Mjc1IDIzLjc0MzkgMS4wNjM0OEMyMy40MzYgMS4xNDYzOCAyMy4xNjUzIDEuMjI4NTggMjIuOTUzOCAxLjMwNzYyQzIyLjg0ODMgMS4zNDcwNiAyMi43NTMgMS4zODcxNyAyMi42NzM1IDEuNDI3NzNDMjIuNTk4MSAxLjQ2NjI4IDIyLjUxOTcgMS41MTQyMyAyMi40NjA2IDEuNTczMjRDMjIuMzI5IDEuNzA0ODUgMjIuMTgyMyAxLjkxMjEgMjIuMjgzOSAyLjE0MDYyQzIyLjMyOTIgMi4yNDIzOSAyMi40MTAzIDIuMzA2NTUgMjIuNDc5MiAyLjM0NTdDMjIuNTQ5IDIuMzg1MjUgMjIuNjI3NyAyLjQxMTE2IDIyLjcwMTkgMi40Mjk2OUMyMi45ODgzIDIuNTAxMjIgMjMuMzk4NiAyLjUgMjMuNjM3NCAyLjVDMjMuNzMyNyAyLjQ5OTkyIDIzLjg5NDYgMi40Njg5MSAyNC4wNzMgMi40Mjk2OUMyNC4yNjM0IDIuMzg3OCAyNC41MDcgMi4zMjgyNSAyNC43ODQ5IDIuMjU3ODFDMjUuMzQwNSAyLjExNjk0IDI2LjA0NyAxLjkyOTA4IDI2Ljc2MzQgMS43NDIxOUMyNy40ODE3IDEuNTU0NzkgMjguMjEzIDEuMzY4MTYgMjguODI1OSAxLjIyODUyQzI5LjQ1MDUgMS4wODYyMSAyOS45MjEyIDEgMzAuMTM3NCAxQzMwLjM2MzMgMS4wMDAwNyAzMC44MTE4IDEuMDU5MDEgMzEuMzc3NiAxLjE1MzMyQzMxLjkzNjIgMS4yNDY0MiAzMi41ODk5IDEuMzcwMzggMzMuMjEzNiAxLjQ5NTEyQzMzLjgzNjcgMS42MTk3NSAzNC40MjkzIDEuNzQ0MzMgMzQuODY1OSAxLjgzNzg5QzM1LjA4MzkgMS44ODQ2MSAzNS4yNjM4IDEuOTIzOTEgMzUuMzg4NCAxLjk1MTE3QzM1LjQ1MDYgMS45NjQ3NyAzNS40OTk5IDEuOTc1MTQgMzUuNTMyOSAxLjk4MjQyQzM1LjU0OSAxLjk4NTk4IDM1LjU2MTYgMS45ODkzNSAzNS41NyAxLjk5MTIxQzM1LjU3NCAxLjk5MjA3IDM1LjU3NzcgMS45OTI3MSAzNS41Nzk4IDEuOTkzMTZMMzUuNTgyNyAxLjk5NDE0QzM1LjcxMDcgMi4wMjIzOCAzNS44MzkyIDEuOTQ2ODkgMzUuODc2NyAxLjgyMTI5QzM1LjkxMzkgMS42OTU1OSAzNS44NDggMS41NjE4MyAzNS43MjUzIDEuNTE1NjJMMzUuNzIyNCAxLjUxNDY1QzM1LjcyMDUgMS41MTM5NSAzNS43MTcxIDEuNTEzMDIgMzUuNzEzNiAxLjUxMTcyQzM1LjcwNTkgMS41MDg4OCAzNS42OTQgMS41MDQ0MyAzNS42Nzk0IDEuNDk5MDJDMzUuNjQ5NSAxLjQ4Nzk2IDM1LjYwNDggMS40NzE3MSAzNS41NDg1IDEuNDUxMTdDMzUuNDM1NiAxLjQwOTkxIDM1LjI3MiAxLjM1MDg3IDM1LjA3MiAxLjI4MDI3QzM0LjY3MTkgMS4xMzkwNSAzNC4xMjIxIDAuOTUwMjQ4IDMzLjUyNTEgMC43NjE3MTlDMzIuOTI4NiAwLjU3MzM2NCAzMi4yODE4IDAuMzgzNzExIDMxLjY4ODIgMC4yNDEyMTFDMzEuMDk5OSAwLjEwMDAyMiAzMC41NDM4IDYuMjgyODJlLTA1IDMwLjEzNzQgMFoiIGZpbGw9IiMxOTI5MzYiLz4KPC9zdmc+Cg=="/>
```

Comment:
seems a bit odd to embed an svg as an image with a data url inside another svg…?